Essentials/src/com/earth2me/essentials/craftbukkit/InventoryWorkaround.java

```diff
@@ -80,6 +80,25 @@ private static Inventory makeTruncatedPlayerInventory(PlayerInventory playerInve
         }
         return addItems(fakeInventory, items);
     }
+    
+    public static Map<Integer, ItemStack> addAllOversizedItems(final Inventory inventory, final int oversizedStacks, final ItemStack... items) {
```

Comment:
Why can't this just be a `Collection<ItemStack>`? That way, you can simply pass in the ArrayList from above without needing to do a cumbersome `toArray(...)`.

Reply:
It could be, but then we should be changing addAllItems and any other usages of it as well. I'm not sure if such a change is appropriate for this pull request or not.